### PR TITLE
Clean up remote configs

### PR DESCRIPTION
**Pull request by bluwave**
refs: MBL-14144
affects: Student, Parent, Teacher
release note: none

### CODE REVIEW

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/fhvnb0i) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Ffhvnb0i)
teacher: [Link](https://rebrand.ly/eoo3ns1) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Feoo3ns1)
parent: [Link](https://rebrand.ly/hhttnel) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fhhttnel)

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Build failed, skipping coverage check
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Student, Parent, Teacher

[MBL-14144](https://instructure.atlassian.net/browse/MBL-14144)

<details><summary>:x: JS tests</summary>


    Summary of all failing tests
     FAIL  src/modules/discussions/details/__tests__/Reply.test.js
      ● DiscussionReplies › renders deleted
    
        Method “props” is meant to be run on 1 node. 0 found instead.
    
          76 |     let tree = shallow(<Reply {...props} />)
          77 |     let webview = tree.find('RichContent')
        > 78 |     expect(webview.props().html.includes('Deleted this reply.')).toEqual(true)
             |                    ^
          79 |   })
          80 | 
          81 |   it('does not show attachments when deleted', () => {
    
          at ShallowWrapper.single (node_modules/enzyme/src/ShallowWrapper.js:1636:13)
          at ShallowWrapper.single [as props] (node_modules/enzyme/src/ShallowWrapper.js:1160:17)
          at Object.props (src/modules/discussions/details/__tests__/Reply.test.js:78:20)
    
    
    Test Suites: 1 failed, 278 passed, 279 total
    Tests:       1 failed, 2441 passed, 2442 total
    Snapshots:   553 passed, 553 total
    Time:        238.451s
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Student/Canvas/Notifications/NotificationsTab.swift [hunk 1]
```diff
@@ -23,13 +23,7 @@ import Core
 
 func NotificationsTab(session: Session) throws -> UIViewController {
```
Comment (vanwagonet) [inline Student/Canvas/Notifications/NotificationsTab.swift]: looks like this no longer `throws`, right?

File: Student/Canvas/Notifications/NotificationsTab.swift [hunk 2]
```diff
@@ -23,13 +23,7 @@ import Core
...
     let title = NSLocalizedString("Notifications", comment: "Notifications tab title")
-    let activityStream: UIViewController
-    if ExperimentalFeature.notifications2.isEnabled {
-        activityStream = ActivityStreamViewController.create()
-    } else {
-        activityStream = try ActivityStreamTableViewController(session: session)
-    }
-    
+    let activityStream: UIViewController = ActivityStreamViewController.create()
     activityStream.title = title
```
Comment (vanwagonet) [inline Student/Canvas/Notifications/NotificationsTab.swift]: I like how you put the title in the controller. Do we still need to set it here too?

File: rn/Teacher/src/routing/register-screens.js [hunk 3]
```diff
@@ -231,13 +231,8 @@ export function registerScreens (store: Store): void {
...
   }
 
-  if (ExperimentalFeature.newPageDetails.isEnabled) {
-    registerScreen('/courses/:courseID/pages/:url', null, store, { deepLink: true })
-    registerScreen('/courses/:courseID/wiki/:url', null, store, { deepLink: true })
-    registerScreen('/groups/:groupID/pages/:url', null, store, { deepLink: true })
-    registerScreen('/groups/:groupID/wiki/:url', null, store, { deepLink: true })
-  } else {
-    registerScreen('/courses/:courseID/wiki/:url', wrap(PageDetails), store, { deepLink: true })
-    registerScreen('/courses/:courseID/pages/:url', wrap(PageDetails), store, { deepLink: true })
```
Comment (vanwagonet) [inline rn/Teacher/src/routing/register-screens.js]: Are we still using PageDetails somewhere else, or can we delete that now?

**Pull request merged**